.github/workflows/sbt-node-snyk.yml

```diff
@@ -117,8 +117,15 @@ jobs:
                   go-version-file: ${{ inputs.GO_VERSION_FILE }}
 
             - name: Snyk monitor
-              run: snyk monitor ${INPUT_DEBUG:+ -d} ${PRUNE_DUPLICATES:+ -p} --all-projects --org="${{ inputs.ORG }}" --exclude="${{ inputs.EXCLUDE }}" --project-tags=commit=${GITHUB_SHA} --
+              run: |
+                snyk monitor \
+                  ${DEBUG_OPTION} \
+                  ${PRUNE_OPTION} \
+                  --all-projects \
+                  --org="${{ inputs.ORG }}" \
+                  --exclude="${{ inputs.EXCLUDE }}" \
+                  --project-tags=commit=${GITHUB_SHA} --
               env:
                   SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
-                  INPUT_DEBUG: ${{ inputs.DEBUG }}
-                  PRUNE_DUPLICATES: ${{ inputs.PRUNE_DUPLICATES }}
+                  DEBUG_OPTION: ${{ inputs.DEBUG == 'true' && '-d' || '' }}
```

Comment:
minor: would updating the type of `DEBUG` to `boolean` (currently `string`) make this any simpler?

Reply:
yeah I thought about that, and figured it would be a good idea, but separate to this